src/types.ts

```diff
@@ -36,6 +37,7 @@ interface AxisProps {
   label?: string;
   showGridLines?: boolean;
   labelStyle?: CSSProperties;
+  tickStyle?: PropertiesHyphen;
```

Comment:
I don't like this because it is different from the other style properties. I'm sure we can avoid it.